### PR TITLE
10263: add rake task to make one-time changes on loan date fields per…

### PR DESCRIPTION
**Pull request by DevneyHamilton**
… client request

### CODE REVIEW

**Comment by DevneyHamilton** [comment]
Wrote this with confused future devs in mind; happy for any feedback.

**Comment by DevneyHamilton** [comment]
manually tested on local copy of prod db.

File: lib/tasks/one_time_changes.rake [hunk 1]
```diff
@@ -0,0 +1,13 @@
+namespace :one_time_changes do
+  desc "A one time task responding to Nov 2019 request to update loan date fields."
+  task adjust_loan_dates: :environment do
+    Loan.all.each do |l|
```
Comment (meltheadorable) [inline lib/tasks/one_time_changes.rake]: I like to use `find_each` in cases like this, IIRC there's some kind of performance benefit to doing so.

**Review by meltheadorable — APPROVED** [review]

File: lib/tasks/one_time_changes.rake (hunk 1 above)
Comment (DevneyHamilton) [reply lib/tasks/one_time_changes.rake]: thanks!

**Comment by DevneyHamilton** [comment]
After this change is deployed to staging, we'll need to manually run `bundle exec rake one_time_changes:adjust_loan_dates RAILS_ENV=staging`. It will also need to be run manually on production after we verify it works on staging.

**Pull request merged**